src/services/config/config.ts

```diff
@@ -1,3 +1,11 @@
 export const config = {
   appEnv: process.env.APP_ENV || "dev",
 };
+
+export const propertyAuthorizedGroups =
+  process.env.APP_ENV === "production"
+    ? ["mmh-data-admin"]
+    : [
+      "mmh-general-user-access",
+      "e2e-testing-development",
+    ];
```

Comment:
This will need to be changed once it's been clarified what user are authorized to access. Currently not set up correctly.